moveit_ros/move_group/src/default_capabilities/get_planning_scene_service_capability.cpp

```diff
@@ -54,7 +54,26 @@ bool move_group::MoveGroupGetPlanningSceneService::getPlanningSceneService(movei
   if (req.components.components & moveit_msgs::PlanningSceneComponents::TRANSFORMS)
     context_->planning_scene_monitor_->updateFrameTransforms();
   planning_scene_monitor::LockedPlanningSceneRO ps(context_->planning_scene_monitor_);
-  ps->getPlanningSceneMsg(res.scene, req.components);
+
+  if (!req.components.components)
```

Comment:
The same comment would be helpful here: "Return all scene components if none are specified"